lib/oas_parser/definition.rb

```diff
@@ -49,9 +49,18 @@ def endpoints
     end
 
     def webhooks
-      return [] unless raw['webhooks']
-      raw['webhooks'].map do |name, definition|
-        OasParser::Webhook.new(self, name, definition)
+      return [] unless raw['webhooks'] || raw['x-webhooks']
+
+      if raw['webhooks']
+        return raw['webhooks'].map do |name, definition|
```

Comment:
@bencgreenberg  @lornajane  can we drop the `return`s? Other than that it looks good

Reply:
👍 done